llama_stack/apis/inference/inference.py

```diff
@@ -157,18 +157,21 @@ class ChatCompletionResponseEvent(BaseModel):
     stop_reason: Optional[StopReason] = None
 
 
+@json_schema_type
 class ResponseFormatType(Enum):
     json_schema = "json_schema"
     grammar = "grammar"
 
 
+@json_schema_type
```

Comment:
also this should end up fixing the jsonschema related issues encountered in kotlin